tests/realtikvtest/scripts/next-gen/run-tests.sh

```
@@ -18,10 +18,91 @@
+    fi
+
+    rm -rf "${install_dir}"
+    mv "${staging_dir}/tiflash" "${install_dir}"
+    rm -rf "${staging_dir}"
+    export TIFLASH_BIN_PATH="${tiflash_bin}"
+}
+
 function main() {
     local make_test_task="$1"
 
     local self_dir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)